man/systemd-vmspawn.xml

```diff
@@ -330,7 +330,7 @@
           <term><option>--private-users=</option><replaceable>UID_SHIFT[:UID_RANGE]</replaceable></term>
 
           <listitem><para>Controls user namespacing under <option>--directory=</option>.
-          If enabled, <citerefentry project='man-pages'><refentrytitle>virtiofsd</refentrytitle><manvolnum>1</manvolnum></citerefentry>
+          If enabled, <citerefentry project='url'><refentrytitle url='https://manpages.debian.org/unstable/qemu-system-common/virtiofsd.1.en.html'>virtiofsd</refentrytitle><manvolnum>1</manvolnum></citerefentry>
```

Comment:
No, we use the Rust version of `virtiofsd`. However, it appears that it doesn't have a man page... So maybe just switch to `<command>virtiofsd</command>`?

Reply:
Done. Am I supposed to wrap the line and if yes how? The [`.vimrc`](https://github.com/systemd/systemd/blob/main/.vimrc) states the line width is 109 characters for XML files, but that has not been followed for this file.

Reply:
It's not a strict limit, so generally OK if some lines exceed that a bit.

Reply:
I tried a thing.